### PR TITLE
JAMES-3491: Jmap push cancellation

### PR DESCRIPTION
**Pull request by LanKhuat**

### CODE REVIEW

File: server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala [hunk 1]
```diff
@@ -567,6 +569,84 @@ trait WebSocketContract {
...
+
+    val mailboxStateChange: String = s"""{"@type":"StateChange","changed":{"$ACCOUNT_ID":{"Mailbox":"$mailboxState"}}}"""
+    val emailStateChange: String = s"""{"@type":"StateChange","changed":{"$ACCOUNT_ID":{"Email":"$emailState"}}}"""
+
+    assertThat(response.toOption.get.asJava)
+      .hasSize(2) // Email create response + no notification message
+      .contains("No notification received")
+      .doesNotContain(mailboxStateChange, emailStateChange)
+  }
+
```
Comment (chibenwa) [inline server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala:649]: What happen if I send a WebSocketPushDisable without sending any WebSocketPushEnable ? Does it fail? Or not?

**Review by chibenwa — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Review by Arsnael — APPROVED** [review]

**Comment by chibenwa** [comment]
`[09024afd6e5ae014ae56d7bbe3936af4e6f59b9a] docker: Error response from daemon: No such container: git-publish-jmap-push-cancelation-b164b3e3-5cc4-46f1-b8fa-6076da9fb659.`

MVN tests :green_apple: 

test this please

**Comment by chibenwa** [comment]
Can you rebase this work?

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**